### DISCUSSION TITLE
On the legend, how to I reduce the space between the icon and the text?

### DISCUSSION THREAD

**Discussion by tnguy271** [GSI-CS-CO/chart-fx#665] [Q&A] [answered]
I’m having trouble customizing the legend. I want to reduce the gap between the legend icons and their corresponding text labels to make the legend more compact. 
![pca](https://github.com/fair-acc/chart-fx/assets/114690375/4d1b2499-458f-4cc3-a899-a055d36abf65)

I’ve tried adjusting CSS properties related to padding and margin:
.chart-legend {
    -fx-padding: 1;
    -fx-spacing: 1;
    -fx-hgap: 1;
    -fx-vgap: 1;
}
However, it doesn't seem to affect the gap of the legend items. 
Thank you for your help!

**Comment by ennerf** [comment, marked as answer]
You are looking for [.chart-legend-item](https://github.com/fair-acc/chart-fx/blob/main/chartfx-chart/src/main/resources/io/fair_acc/chartfx/chart.scss#L219-L222).

Running [ScenicView](https://github.com/JonathanGiles/scenic-view) would show you the whole hierarchy of your application. It can be run in a standalone process or [embedded](https://mvnrepository.com/artifact/net.raumzeitfalle.fx/scenic-view/11.0.2) into your application.

**Reply by wirew0rm** [reply]
Thanks Florian for looking up the correct css selektor :+1:  Scenic view is also included by default into our sample application where it can be launched from the initial view.

**Reply by tnguy271** [reply]
Thanks all, it works!
